tests/runner/test_sequential_runner.py

```diff
@@ -278,3 +279,77 @@ def test_confirms(self, mocker, pipeline, is_async):
         catalog = DataCatalog(data_sets={"ds1": fake_dataset_instance})
         SequentialRunner(is_async=is_async).run(pipeline, catalog)
         fake_dataset_instance.confirm.assert_called_once_with()
+
+
+@pytest.fixture
+def resume_scenario_pipeline():
```

Comment:
Should we define this in conftest where the other pipeline fixtures are? Would be good to give it a more descriptive name also like `X_shaped_pipeline` (c.f. `branchless_pipeline`) and maybe a short docstring describing or picturing the structure (like `branchless_no_input_pipeline` has).

Reply:
I think you're right about this. I modified the names and moved my fixtures to conftest. I also moved the remaining fixtures in that file to conftest, in the spirit of leaving the codebase cleaner than when I found it. The remaining fixtures were from 2019 and came from a single contributor, while the more modern fixtures were all defined in conftest and were from multiple contributors, which is a good sign to me that this is the better approach.

These changes are made in [069a6fe](https://github.com/kedro-org/kedro/pull/1795/commits/069a6feede3bf53d7688aeb1a1511d58b9c3d3e9).

Reply:
Perfect, thank you! Definitely sounds like an improvement to the codebase.